major-mode-hydra.el

```diff
@@ -75,7 +75,7 @@ Overwrite existing hydra if OVERWRITE-P is t, otherwise add new heads to it."
          (title (when (functionp major-mode-hydra-title-generator)
                   (funcall major-mode-hydra-title-generator mode)))
          (body (-> body
-                   (lax-plist-put :hint nil)
+                   (compat-call plist-put :hint nil #'equal)
```

Comment:
Sorry for the super late response. Work has been busy and I kinda forgot about this PR. This bit breaks because of the `->` macro. It gets expanded to `(compat-call body plist-put :hint nil #'equal)` which is wrong. I'll merge your PR and fix it on `master`.